lib/index.js

```diff
@@ -16,29 +16,62 @@ let pomodoro = new Pomodoro();
 
 // pomodoro.getStorage();
 
-$startButton.on('click', function() {
-  if (pomodoro.timers.length >= 2) return;
+function addTimers() {
   pomodoro.addTimer();
-  pomodoro.addTimer(0.5,'unstarted', 'break');
-  pomodoro.timers[0].startTimer();
-  pomodoro.tick();
+  pomodoro.addTimer(0.1,'pending', 'break');
+}
+
+function startActiveTimer() {
+  for(var i = 0; i < pomodoro.timers.length; i ++) {
+    if(pomodoro.timers[i].currentState === 'unstarted') {
+      pomodoro.timers[i].startTimer();
+      console.log(pomodoro.timers);
+      console.log(pomodoro.timers[i]);
+      pomodoro.tick(i);
+    }
+  }
+}
+
+$startButton.on('click', function() {
+  if (pomodoro.timers.length !== 2) {
+    addTimers();
+    startActiveTimer();
+  } else {
+    startActiveTimer();
+  }
 });
 $pauseButton.on('click', function() {
-  // for (var i = 0; i < pomodoro.timers.length - 1; i ++) {
-  //
-  // }
-  if (pomodoro.timers[0].currentState !== 'running') return;
-  pomodoro.timers[0].pauseTimer();
+  for(var i = 0; i < pomodoro.timers.length; i ++) {
+    if(pomodoro.timers[i].currentState === 'running') {
+      pomodoro.timers[i].pauseTimer();
+    }
+  }
 });
-$resumeButton.on('click', function() {
+
+function checkBeforeResume(n) {
   if (pomodoro.timers.length === 0) return;
-  if (pomodoro.timers[0].currentState !== 'paused') return;
-  pomodoro.timers[0].resumeTimer();
-  pomodoro.tick();
+  if (pomodoro.timers[n].currentState !== 'paused') return;
+}
+
+$resumeButton.on('click', function() {
+  for(var i = 0; i < pomodoro.timers.length; i ++) {
+    if(pomodoro.timers[i].currentState === 'paused') {
```

Comment:
reason we did this was so that you couldn't resume something that is not paused.  in this approach, start and resume have 2 specific responsibilities.  Start only starts a new game and resume only resumes when the timer is paused.